renderer/components/Activity/ActivityActions/ActivityActions.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { injectIntl, intlShape } from 'react-intl'
+import { injectIntl } from 'react-intl'
 import { Flex } from 'rebass'
 import { Card } from 'components/UI'
 import ActivityFilter from './ActivityFilter'
 import ActivityRefresh from './ActivityRefresh'
 import ActivitySearch from './ActivitySearch'
 import messages from './messages'
+import { intlShape } from '@zap/i18n'
```

Comment:
import should go before relative imports (this comment applies to the whole PR)

Reply:
updated